cms/core/blocks/stream_blocks.py

```diff
@@ -1,31 +0,0 @@
-from typing import ClassVar
-
-from wagtail.blocks import RichTextBlock, StreamBlock
-from wagtail.embeds.blocks import EmbedBlock
-from wagtail.images.blocks import ImageChooserBlock
-from wagtailmath.blocks import MathBlock
-
-from cms.core.blocks import (
-    DocumentsBlock,
-    HeadingBlock,
-    ONSEmbedBlock,
-    PanelBlock,
-    RelatedLinksBlock,
-)
-
-
-class CoreStoryBlock(StreamBlock):
```

Comment:
this is used by `InformationPage`. it is purposefully simpler

Reply:
I've skipped `standard_pages/models.py` because I was still working on the Methodology page there. Are there any other differences than the `Media` group that I'm not aware of? and was that a requirement or preference? I'm just not sure if it's worth having so much duplicated code for the sake of a slightly more minimalistic UI.

Reply:
@kacperpONS this makes the information page StreamField unnecessarily complex. If we don't have a title, then the individual blocks should be top level.

Could you please keep the two separate? At least until we are 100% certain they will be the same everywhere

Reply:
Separated them in [96e07a6](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/96e07a6d7def5cbeaf56c84a166e13aacb77a38e). 
Question - is it not worth wrapping the content of `CoreStoryBlock` in a `StructBlock` so that the structure of having one `heading` per block is enforced?
That would be similar to how it's done for the `SectionStoryBlock` with the intermediary `class SectionBlock(StructBlock)`
Currently you can create a `CoreStoryBlock`  that contains just a header, or text with no header; or is that intentional since it's supposed to be simple? (and not as restrictive?)